psutil/_psutil_windows.c

```diff
@@ -2917,7 +2781,7 @@ psutil_net_if_addrs(PyObject *self, PyObject *args) {
 #if PY_MAJOR_VERSION >= 3
                 py_address = PyUnicode_FromString(buff);
 #else
-                py_address = PyString_FromString(buff);
+                py_address = PyBytes_FromString(buff);
```

Comment:
This was originally a separate change.  PyString_AS_STRING vanished from Python 3.4(?) and I thought it would make sense to use PyBytes everywhere instead.  I can revert this.